guides/common/modules/proc_creating-a-content-view.adoc

```diff
@@ -10,23 +10,31 @@ While you can stipulate whether you want to resolve any package dependencies on
 For more information, see xref:Resolving_Package_Dependencies_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create Content View*.
 . In the *Name* field, enter a name for the view.
 {Project} automatically completes the *Label* field from the name you enter.
 . In the *Description* field, enter a description of the view.
+. In the *Type* field, choose a *Component content view* or a *Composite content view*.
 . Optional: if you want to solve dependencies automatically every time you publish this Content View, select the *Solve Dependencies* check box.
 Dependency solving slows the publishing time and might ignore any Content View filters you use.
 This can also cause errors when resolving dependencies for errata.
-. Click *Save* to create the Content View.
-. In the *Repository Selection* area, select the repositories that you want to add to your Content View, then click *Add Repositories*.
-. Click *Publish New Version* and in the *Description* field, enter information about the version to log changes.
-. Click *Save*.
-. Optional: to force metadata regeneration on Yum repositories, from the *Actions* list for your Content View versions, select *Regenerate Repository Metadata*.
+. Click *Create Content View*.
+
+.Component Content View Steps
+. Click *Create Content View* to create the Content View.
+. In the *Repositories* tab, select the repository from the *Type* list that you want to add to your Content View, select the checkbox next to the available repositories you want to add, then click *Add repositories*.
+. Click *Publish new version* and in the *Description* field, enter information about the version to log changes.
+. Optional: You can enable a promotion path by clicking *Promote* to *Select a lifecycle environment from the available promotion paths to promote new version*.
+. Click *Next*.
+. On the *Review* page, you can review the environments you are trying to publish.
+. Click *Finish*.
+. Optional: In the *Versions* tab after publishing, you can *Promote*, *Remove*, and *Delete* versions of the Content View under the options menu.
+[NOTE]
+*Remove* and *Delete* are similar but the *Delete* option will delete an entire Content View and the versions associated with that lifecycle environment. The *Remove* option allows you to choose which version you want removed from the lifecycle environment.
```

Comment:
One sentence per line rule.

Reply:
We don't use future tense as per IBM style.

Reply:
We don't write "allows you" - as per Anthropomorphism section in the IBM Style guide.

Reply:
s/"want removed"/"want to remove"

Reply:
Noted and changed per request.